protostar/starknet/__init__.py

```diff
@@ -1,3 +1,5 @@
+from starkware.starknet.business_logic.state.state import CachedState as StarknetState
```

Comment:
Moreover it may be confused with `starkware.starknet.testing.state.StarknetState` which is a totally different thing

Reply:
`CachedState` doesn't make much sense from Protostar perspective.

Reply:
Even though it is the case, it is still hard to follow. It is better to use one name for one thing, even though it is imperfect. It is easier to know what we are talking about. Something like this will cause a lot of clarification questions in tech discussions "Do we talk about renamed cached state?"

Reply:
this is also the term that will continue to be used in rust implementation, so it will be easier to reason about the migration to rust, and `legacy <-> new codebase` relation when we get to it.